app/src/main/java/io/homeassistant/companion/android/database/widget/MultiWidgetEntity.kt

```diff
@@ -0,0 +1,17 @@
+package io.homeassistant.companion.android.database.widget
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import io.homeassistant.companion.android.widgets.multi.MultiWidget.Orientation
+import io.homeassistant.companion.android.widgets.multi.elements.MultiWidgetElementEntity
+
+@Entity(tableName = "multi_widgets")
+data class MultiWidgetEntity(
+    @PrimaryKey
+    val id: Int,
+    @ColumnInfo(name = "elements")
+    val elements: List<MultiWidgetElementEntity>,
+    @ColumnInfo(name = "orientation")
+    val orientation: Orientation
```

Comment:
I specifically left it as an enum in case we ended up creating an grid-style, and for general readability.  At the moment, technically it maps Orientation into a Boolean, but that could change later if a grid is implemented.